library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+                **** Myoclonic-atonic-seizure <nowiki>[A generalized seizure type with a myoclonic jerk leading to an atonic motor component. This type was previously called myoclonic astatic. Definition from ILAE 2017 Classification of Seizure Types Expanded Version. (Source: Fisher ea 2017, Table 2.)]</nowiki>
+                **** Myoclonic-tonic-clonic-seizure <nowiki>[One or a few jerks of limbs bilaterally, followed by a tonic clonic seizure. The initial jerks can be considered to be either a brief period of clonus or myoclonus. Seizures with this characteristic are common in juvenile myoclonic epilepsy. Definition from ILAE 2017 Classification of Seizure Types Expanded Version. (Source: Fisher ea 2017, Table 2.)]</nowiki>
+                **** Tonic-clonic-seizure <nowiki>[A sequence consisting of a tonic followed by a clonic phase. Definition from ILAE 2017 Classification of Seizure Types Expanded Version. (Source: Fisher ea 2017, Table 2.)]</nowiki>
+                **** Automatism-seizure <nowiki>[A more or less coordinated motor activity usually occurring when cognition is impaired and for which the subject is usually (but not always) amnesic afterward. This often resembles a voluntary movement and may consist of an inappropriate continuation of preictal motor activity. Definition from ILAE 2017 Classification of Seizure Types Expanded Version. (Source: Fisher ea 2017, Table 2.)]</nowiki>
+                **** Hyperkinetic-seizure <nowiki>[(Source: Fisher ea 2017, Table 2.)]</nowiki>
+                **** Epileptic-spasm <nowiki>[A sudden flexion, extension, or mixed extension flexion of predominantly proximal and truncal muscles that is usually more sustained than a myoclonic movement but not as sustained as a tonic seizure. Limited forms may occur: Grimacing, head nodding, or subtle eye movements. Epileptic spasms frequently occur in clusters. Infantile spasms are the best known form, but spasms can occur at all ages. Definition from ILAE 2017 Classification of Seizure Types Expanded Version. (Source: Fisher ea 2017, Table 2.)]</nowiki>
+            *** Nonmotor-seizure <nowiki>{suggestedTag=Duration}[Focal or generalized seizure types in which motor activity is not prominent. Definition from ILAE 2017 Classification of Seizure Types Expanded Version. (Source: Fisher ea 2017, Table 2; Duration tag from Beniczky ea Table 13.)]</nowiki>
+                **** Behavior-arrest-seizure <nowiki>[Arrest (pause) of activities, freezing, immobilization, as in behavior arrest seizure. A focal behavior arrest seizure shows arrest of behavior as the prominent feature of the entire seizure. Definition from ILAE 2017 Classification of Seizure Types Expanded Version. (Source: Fisher ea 2017.)]</nowiki>
```

Comment:
Some of the seizures in this section have a suggestedTag=Duration and others don't.  Should they all have the suggestedTag=Duration?

Reply:
suggested-tags are inherited, so they all have duration